app/src/main/java/net/squanchy/support/lang/OptionalEnums.java

```diff
@@ -0,0 +1,16 @@
+package net.squanchy.support.lang;
+
+import android.text.TextUtils;
+
+import net.squanchy.eventdetails.domain.view.ExperienceLevel;
+
+public final class OptionalEnums {
+
+    public static Optional<ExperienceLevel> from(String rawLevel) {
```

Comment:
@rock3r I used `TextUtils.isEmpty()` mostly to keep on using the current data base, in which we get a lot of empty strings. If we use null, it will crash unless we fix the data to use the values we expect, but I'll fix this right away